docs/tutorials/mastering.md

```diff
@@ -1,60 +1,624 @@
 
+## 1. Prepare Host
 
-## Sample Values
-
-For the purpose of this guide, the following values will be used.
+A machine running Ubuntu 22.04 with the following resources:
+  - At least one NIC with internet access
+  - 8 cores
+  - 24 GB RAM
+  - 150 GiB disk  
 
 ### Networks
 
+The following networks will be used to create test environment.
+
 | Name | Subnet | Gateway IP |
 | ---- | ------ | ---------- |
 | `management` | 10.201.0.0/24 | 10.201.0.1 |
 | `access`     | 10.202.0.0/24 | 10.202.0.1 |
 | `core`       | 10.203.0.0/24 | 10.203.0.1 |
 | `ran`        | 10.204.0.0/24 | 10.204.0.1 |
 
+
+Create local network bridges to be used by LXD by adding below configuration under `/etc/netplan/50-sdcore-networks.yaml`.
+
+```console
+cat << EOF | sudo tee /etc/netplan/50-sdcore-networks.yaml
+# /etc/netplan/50-sdcore-networks.yaml
+network:
+  bridges:
+    mgmt-br:
+      addresses:
+        - 10.201.0.14/24
+      routes:
+        - to: default
+          via: 10.201.0.1
+          metric: 110
+    access-br:
+      addresses:
+        - 10.202.0.14/24
+      routes:
+        - to: 10.204.0.0/24
+          via: 10.202.0.1
+    core-br:
+      addresses:
+        - 10.203.0.14/24
+      routes:
+        - to: default
+          via: 10.203.0.1
+          metric: 203
+    ran-br:
+      addresses:
+        - 10.204.0.14/24
+      routes:
+        - to: 10.202.0.0/24
+          via: 10.204.0.1
+
+  version: 2
+EOF
+```
+
+```console
+sudo netplan apply
+```
+
+### Install and Configure LXD
+
+Install LXD:
+```console
+sudo snap install lxd
+```
+
+```console
+cat << EOF | sudo tee ~/preseed.yaml
+config: {}
+networks: []
+storage_pools:
+- config:
+    size: 150GiB
+    source: /var/snap/lxd/common/lxd/disks/default.img
+    zfs.pool_name: default
+  description: ""
+  name: default
+  driver: zfs
+profiles:
+- config: {}
+  description: Default LXD profile
+  devices:
+    root:
+      path: /
+      pool: default
+      type: disk
+  name: default
+projects:
+- config:
+    features.images: "true"
+    features.networks: "true"
+    features.networks.zones: "true"
+    features.profiles: "true"
+    features.storage.buckets: "true"
+    features.storage.volumes: "true"
+  description: Default LXD project
+  name: default
+EOF
+```
+
+```console
+lxd init --preseed < ~/preseed.yaml 
+```
+
+### Install and configure Multipass
+
+Install Multipass and set LXD as local driver:
+
+```console
+sudo snap install multipass
+sudo multipass set local.driver=lxd
+```
+Wait a few seconds if you get the output: `set failed: cannot connect to the multipass socket` and retry to set local driver again.
+
+Check if local driver is set to lxd properly.
+```console
+$ sudo multipass get local.driver
+lxd
+```
```

Comment:
Is this necessary? Some of these checks seem superfluous and make the tutorial longer.

Reply:
we can get rid of it

Reply:
Done